components/Dialogs/examples/DialogsAlertCustomizationViewController.swift

```diff
@@ -270,6 +286,13 @@ class DialogsAlertCustomizationViewController: MDCCollectionViewController {
 
 }
 
+// app specific elevation constants
+extension ShadowElevation {
+  static var appDialog: ShadowElevation {
```

Comment:
Please append a prefix unique to this example.  Maybe dialogsAlert_?

Reply:
Done